xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxUtils.cpp

```diff
@@ -82,3 +86,12 @@ DemuxPacket* CDVDDemuxUtils::AllocateDemuxPacket(unsigned int iDataSize, unsigne
     ret->cryptoInfo = std::shared_ptr<DemuxCryptoInfo>(new DemuxCryptoInfo(encryptedSubsampleCount));
   return ret;
 }
+
+void CDVDDemuxUtils::StoreSideData(DemuxPacket *pkt, AVPacket *src)
+{
+  AVPacket avPkt;
+  av_init_packet(&avPkt);
+  av_packet_copy_props(&avPkt, src);
```

Comment:
src might be null, which directly crashes: https://ffmpeg.org/doxygen/3.1/avpacket_8c_source.html#l00531